src/modules/rewards/components/RewardsActivitiesListItem.tsx

```diff
@@ -107,57 +127,57 @@ const RewardsActivitiesListItem: FC<RewardActivitiesListItemProps> = ({ userId,
-                      {activity.activityTitle}
-                    </Text>
-                    {/* </Link> */}
+                    {resolveActivityTitle(activity.activityTitle)}
                   </Skeleton>
 
                   {!!activity.expiryType && (
                     <Box display="flex">
-                      <Lozenge size="small">Expires in {getUpdatedExpiryTime(activity.expiryType)} days</Lozenge>
+                      <Lozenge css={css`text-transform:uppercase;`} size="small">Expires in {getUpdatedExpiryTime(activity.expiryType)} days</Lozenge>
                     </Box>
                   )}
                 </Box>
-                <Skeleton isLoading={isLoading}>
-                  <Text variant="h5-regular" color="gray-500">
-                    {activity.activityDesc}
-                  </Text>
-                </Skeleton>
+
+                {/* We no need to show the Description when the title is discord and twitter according to the design */}
+                {activity.activityType !== 'follow_push_on_discord' && activity.activityType !== 'follow_push_on_twitter' && (
+                  <Skeleton isLoading={isLoading}>
+                    <Text variant="h5-regular" color="gray-500">
+                      {activity.activityDesc}
+                    </Text>
+                  </Skeleton>
+                )}
+
               </Box>
 
               {/* Rewards Points */}
-              <Box display="flex" minWidth="160px" flexDirection="row" gap="s2" alignItems="center">
-                <Skeleton isLoading={isLoading} height='32px'>
+              <Box display="flex" minWidth="200px" flexDirection="row" gap="s2" alignItems="center">
+                <Skeleton isLoading={isLoading} height="32px">
                   <RewardsBell width={32} height={32} />
                   <Text
                     variant="h4-semibold"
                     color={{ light: 'gray-1000', dark: 'gray-100' }}
-                    css={css`
-                    margin-right: 24px;
-                  `}
                   >
                     {activity.points?.toLocaleString()} points
                   </Text>
                 </Skeleton>
               </Box>
             </Box>
 
-
-
             {/* Buttons Logic */}
 
             {usersSingleActivity?.status === 'COMPLETED' && (
-              <Box display="flex" alignItems={{ ml: 'flex-start', initial: 'center' }} flexDirection="column" minWidth="100px">
+              <Box
+                display="flex"
+                alignItems={{ ml: 'flex-start', initial: 'center' }}
+                flexDirection="column"
+                minWidth="100px"
+              >
                 <Skeleton width="100%" isLoading={isLoading}>
                   <Button
                     variant="tertiary"
                     size="small"
                     css={css`
-                 width: 100%;
-               `}
+                      width: 100%;
+                    `}
```

Comment:
not required

Reply:
This is needed because button parent has a minWidth of 100px. And Button has no prop to pass related to width. Extra Small button variant is required but their is no way to pass width.

According to design -> https://www.figma.com/design/FvJL0JWHlsVohsHTERB1Lh/Push-Design-Foundations-[Future]?node-id=3152-5415&m=dev